projects/media-viewer/src/lib/viewers/image-viewer/image-viewer.component.html

```diff
@@ -1,9 +1,12 @@
 <mv-comment-set-toggle *ngIf="enableAnnotations"></mv-comment-set-toggle>
-<div id="viewer-wrapper"
+<div dragScroll [scrollAxis]="'horizontal'" [enableScroll]="enableDragScroll"
```

Comment:
do we need two inputs here? would it not work by binding the value of the drag toggle with the scrollAxis (which I think in this case should be dragX), so if it's on dragX would be true otherwise it's false